phpBB/assets/javascript/core.js

```diff
@@ -1654,7 +1654,7 @@ $(function() {
 
 	phpbb.registerPageDropdowns();
 
-	$('#color_palette_placeholder').each(function() {
+	$('[data-orientation]').each(function() {
```

Comment:
Use `data-color-palette`?

Reply:
Currently, there is no such data property. As per hanakin's comments, he would like to see it that way in the future, but this suffices now.

Reply:
Data attributes aren't predefined which is why I think this would make sense. See also: https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes

Reply:
Yeah, I am aware how data attributes work. I just think we don't need to add an extra data attribute just to make this one selector easier to read. User needs to define `data-orientation` anyway, color palette wouldn't work without it.

Reply:
hes not saying add it. what we are saying is rename orientation to color-pallette it would require one extra line of js be change and the html being changed which is how I would like to see it, however as this is 3.2 I'm ok with it being as is since there might be some extensions that use it that would break

Reply:
So either I leave it as it is or change the base branch to 3.3 and rename it?

Reply:
I would say leave it as is for 3.2 and make a new pr for 3.3 @marc1706 @Nicofuma

Reply:
https://github.com/phpbb/phpbb/pull/5136